regression/esbmc-solidity/polymorphism_1/polymorphism.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0
+
+// Different contracts with the same function name. 
+pragma solidity >=0.5.0;
+
+contract A {
+
+    function poly() public pure returns (uint8) {
+        return 21;
+    }
+}
+
+contract B {
+    function poly() public pure returns (uint8) {
+        return 42;
+    }
+}
+
+contract C {
+    function test_poly() public {
+        A contract_A = new A();
```

Comment:
This test case(TC) doesn't cover polymorphism. Please update the name for this TC.

Reply:
also for other TCs named `polymorphism_*`

Reply:
Will do.